src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+    <metadata>
+        <id>VS.Redist.Common.NetCore.Toolset.$ARCH$</id>
+        <version>$MAJOR_VERSION$.$MINOR_VERSION$</version>
```

Comment:
I'm concerned about this version here as nuget will add a patch version after, making the package name VS.Redist.Common.AspNetCore.TargetingPack.x64.3.0.0.nupkg and VS.Redist.Common.NetCore.Toolset.x64.3.0.0.nupkg. Should we have patch/preview version numbers here @johnbeisner ?